### PR TITLE
Jcottriel sdi updating content events

### PR DESCRIPTION
**Pull request by jcottriel-further**

### CODE REVIEW

File: docs/events/content/view_content.md [hunk 1]
```diff
@@ -6,29 +6,31 @@ title: View Content
...
 Fire whenever a page whose main purpose is to serve content (e.g. an article or blog post) loads.
 
+This event is to be used to understand when content is being viewed without any association to a "[Content List](../../events/content/view_content_listing.md)". If you are looking to report on how much a piece of content was engaged with and want/need to know the list information, you must use "[select_content](../../events/content/view_content_listing.md)" in your GA4 report as your event and NOT "view_content". This is because "view_content" will not have associated list information.
+
 ## Javascript Code
 
 ```js
 // When:
-// User views an article, blog post, or other content-focused page
+// User views an article, blog post, or other content-focused pages
```
Comment (tyssejc) [inline docs/events/content/view_content.md]: nyet — if `view_content` fires, they're only looking at one page

File: docs/events/content/view_content_listing.md [hunk 2]
```diff
@@ -6,29 +6,33 @@ title: View Content Listing
...
+Pairs with the following event:
+- [Select Content](../../events/content/select_content.md)
+
+
 ## Javascript Code
 
 ```js
 // When:
-// User views an article, blog post, or other content-focused page
+// User views an article, blog post, or other content-focused pages
```
Comment (tyssejc) [inline docs/events/content/view_content_listing.md]: suggested: User views **a listing of** articles, article cards, etc.

File: docs/events/content/select_content.md [hunk 3]
```diff
@@ -6,6 +6,11 @@ title: Select Content
 
 Fire whenever a user clicks on a content link found in a list that forwards the user to the content page. List types include card collections, search results, carousels, and other similar components. 
 
+This event is to be used to understand how content is being engaged with in regard to the lists they are a part of. If you are looking to report on how much a piece of content was engaged with and want/need to know the list information, you must use "select_content"  in your GA4 report as your event and NOT "[view_content](../../events/content/view_content.md)". This is because "[view_content](../../events/content/view_content.md)" will not have associated list information.
```
Comment (tyssejc) [inline docs/events/content/select_content.md]: good callout, but the audience for the spec is developers, not site owners/analysts. this is better in an SDR or something imo. I do like the idea of giving developers context around what type of reporting an event enables

**Review by tyssejc — CHANGES_REQUESTED** [review]
wow you were busy! minor changes requested — also I think we would benefit from a larger discussion around adding context for developers so they know _why_ they're being asked to implement these events, but that can be a separate effort (not part of this pull)

File: docs/events/content/select_content.md (hunk 3 above)
Comment (jcottriel-further) [reply docs/events/content/select_content.md]: cool. so remove?

File: docs/events/content/view_content.md (hunk 1 above)
Comment (jcottriel-further) [reply docs/events/content/view_content.md]: cool, so leave as "page"?

File: docs/events/content/view_content_listing.md (hunk 2 above)
Comment (jcottriel-further) [reply docs/events/content/view_content_listing.md]: cool, ill update this and others where applicable

File: docs/events/content/select_content.md (hunk 3 above)
Comment (tyssejc) [reply docs/events/content/select_content.md]: yes, remove for now. maybe we can open a discussion about how to tie events devs are implementing to what kind of reporting those events/params enable, so they can better understand what they're being asked to do.

File: docs/events/content/view_content.md (hunk 1 above)
Comment (jcottriel-further) [reply docs/events/content/view_content.md]: reverted

File: docs/events/content/view_content_listing.md (hunk 2 above)
Comment (jcottriel-further) [reply docs/events/content/view_content_listing.md]: updated

File: docs/events/content/select_content.md (hunk 3 above)
Comment (jcottriel-further) [reply docs/events/content/select_content.md]: reverted

**Review by tyssejc — APPROVED** [review]
Approved with some minor changes:
- on view_content, hide the note about using select_content for engagement in a collapsible admonition 
- on view_content_listing, update examples, add note about multiple content listings

**Pull request merged**